index.js

```diff
@@ -336,6 +356,28 @@ Atrium.Client.prototype.listMemberTransactions = function (request = {}) {
   return this._fetchUtility(`users/${request.params.userGuid}/members/${request.params.memberGuid}/transactions` + params, 'GET');
 };
 
+Atrium.Client.prototype.verifyMember = function (request = {}) {
+  return this._fetchUtility(`users/${request.params.userGuid}/members/${request.params.memberGuid}/verify`, 'POST');
+};
+
+Atrium.Client.prototype.listAccountNumbers = function (request = {}) {
```

Comment:
Since there are separate endpoints for listing account numbers by account or member, this method can be split into two, avoiding the need for the conditional altogether.